Makefile

```diff
@@ -1,101 +0,0 @@
-#  Illinois Open Source License
-#  #  
-#  University of Illinois/NCSA
-#  #  Open Source License
-#  
-#  #  Copyright � 2009,    University of Illinois.  All rights reserved.
-#  
-#  #  Developed by:
-#  
-#  #  Innovative Systems Lab  
-#  National Center for Supercomputing Applications  
-#  #  http://www.ncsa.uiuc.edu/AboutUs/Directorates/ISL.html
-#  
-#  #  Permission is hereby granted, free of charge, to any person obtaining a copy of 
-#  this software and associated documentation files (the "Software"), to deal with 
-#  #  the Software without restriction, including without limitation the rights to use,
-#  copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the 
-#  #  Software, and to permit persons to whom the Software is furnished to do so, subject
-#  to the following conditions:
-#  #  
-#  * Redistributions of source code must retain the above copyright notice, this list 
-#  #  of conditions and the following disclaimers.
-#
-##  * Redistributions in binary form must reproduce the above copyright notice, this list
-#  of conditions and the following disclaimers in the documentation and/or other materials
-#  #  provided with the distribution.
-#
-##  * Neither the names of the Innovative Systems Lab, the National Center for Supercomputing
-#  Applications, nor the names of its contributors may be used to endorse or promote products
-#  #  derived from this Software without specific prior written permission.
-#  
-#  #  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
-#  INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
-#  #  PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE CONTRIBUTORS OR COPYRIGHT HOLDERS BE
-#  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT 
-#  #  OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
-#  DEALINGS WITH THE SOFTWARE.
-#  #
-
-#default, assume you are using NVIDIA GPUs
-#default target is cuda_memtest
-#TARGET=cuda_memtest ocl_memtest
-TARGET=cuda_memtest
-CUDA_INSTALL_PATH ?=/usr/local/cuda
-
-#if you are using AMD GPUs, uncomment the following line and set the install path correctly 
-#TARGET=ocl_memtest
-AMD_INSTALL_PATH ?=/usr/local/ati-stream-sdk-v2.1-lnx64/
```

Comment:
wow, I am old enough to remember ATI stream.

Reply:
I did use it in the good old times xD